components/infobox/wikis/stormgate/infobox_unit_custom.lua

```diff
@@ -180,6 +191,13 @@ function CustomUnit:setLpdbData(args)
 		image = args.image,
 		imagedark = args.imagedark,
 		extradata = mw.ext.LiquipediaDB.lpdb_create_json{
```

Comment:
could you check how long the stringified extradata of a fully filled infobox is?
like the number you would get from `mw.logObject(string.len(mw.ext.LiquipediaDB.lpdb_create_json(extradata)))`

Reply:
With `{name=Weaver,built=Conclave,built_link=Conclave,hotkey=C,macro_key=W,macro_key2=C,faction=i,type=Ground,building_requirement='Ritual Chamber',luminite=100,therium=150,supply=4,buildtime=0.5,charge_time=35,sight=10,speed=2,passive='Detector, High Sight, Soul Ripper, Shroudwalk',health=580,extra_health=145,armor_type='Biological, Light',armor=0,introduced='E45737',deprecated='F55366',veterancybonushealth='50, 50, 50',veterancybonusdamage='',veterancybonusattackspeed='10%, 10%, 10%',veterancyxp='1000, 2000, 3000'}` its 671. Technically that could still increase a bit with info (e.g. multiple buildings listed in `building_requirement`)